cpp/src/arrow/csv/parser-benchmark.cc

```diff
@@ -28,44 +28,36 @@
 namespace arrow {
 namespace csv {
 
-static std::string BuildQuotedData(int32_t num_rows = 10000) {
-  std::string one_row = "abc,\"d,f\",12.34,\n";
-  std::stringstream ss;
-  for (int32_t i = 0; i < num_rows; ++i) {
-    ss << one_row;
-  }
-  return ss.str();
-}
+// Can't have static str.
+const char* one_row = "abc,\"d,f\",12.34,\n";
+const char* one_row_escaped = "abc,d\\,f,12.34,\n";
 
-static std::string BuildEscapedData(int32_t num_rows = 10000) {
-  std::string one_row = "abc,d\\,f,12.34,\n";
+size_t num_rows = (1024 * 8) / strlen(one_row);
```

Comment:
That's extremely small.

Reply:
Wanted to fit in L1, do you suggest a better size? Local tests shows not much difference by increasing this.

Reply:
At least 64 kiB. Fitting in L1 is not really a goal here.

Reply:
I'm not saying that fitting in L1 is the goal, I meant that some benchmarks (e.g. the kernels), are more limited by the memory bandwidth so lowering the input does show the "maximal" rate. In this case, it doesn't change anything so I'll leave it to 64kb.

Note that my process was to go through all benchmarks and lower the inputs to get a faster running suite. This change is a collateral damage.